Loadouts/csat_ctar_pacmech_apex.hpp

```diff
@@ -0,0 +1,298 @@
+};
+class potato_msv_AR: potato_msv_rifleman {// AR
+  weapons[] = {AR};
+  magazines[] = {AR_MAG,PISTOL_MAG,BASE_GRENADES};
+  handguns[] = {PISTOL};
+  attachments[] = {};
+  opticChoices[] = {};
+};
+class potato_msv_g: potato_msv_rifleman {// Grenadier
+  magazines[] += {AT_MAG};
+  launchers[] = {AT};
+};
+class potato_msv_ag: potato_msv_rifleman {// Assistant Grenadier
+  magazines[] += {AT_MAG};
+};
+class potato_msv_sm: potato_msv_rifleman {// Medic
+  magazines[] = {CARBINE_MAG,MEDIC_GRENADES};
+  backpackItems[] = {MEDIC_MEDICAL};
+};
+class potato_msv_marksman: potato_msv_rifleman {// Squad Marksman
+  weapons[] = {MARKSMAN};
+  magazines[] = {MARKSMAN_MAG};
+  attachments[] = {"optic_DMS"};
+  opticChoices[] = {"optic_DMS","optic_DMS_ghex_F"};
+};
+class Fic_Spotter: potato_msv_rifleman {
+  linkedItems[] += {RANGE_FINDER};
+};
+class potato_msv_mmgg: potato_msv_AR {// MMG
+  backpack[] = {"B_Carryall_ghex_F"};
+  weapons[] = {MMG};
+  magazines[] = {MMG_MAG,PISTOL_MAG,BASE_GRENADES};
+};
+class potato_msv_mmgag: Fic_Spotter {// MMG Spotter/Ammo Bearer
+  backpack[] = {"B_Carryall_ghex_F"};
+  backpackItems[] += {MMG_MAG};
+};
+class potato_msv_matg: potato_msv_rifleman {// MAT Gunner
+  backpack[] = {"B_Carryall_ghex_F"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG};
+  items[] += {BASE_MEDICAL};
+  launchers[] = {MAT};
+};
+class potato_msv_matag: Fic_Spotter {// MAT Spotter/Ammo Bearer
+  backpack[] = {"B_Carryall_ghex_F"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG2};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_msamg: potato_msv_rifleman {// SAM Gunner
+  SAM_GEAR("B_Carryall_ghex_F", SAM_MAG)
+  launchers[] = {SAM};
+};
+class potato_msv_msamag: Fic_Spotter {// SAM Spotter/Ammo Bearer
+  SAM_GEAR("B_Carryall_ghex_F", SAM_MAG)
+};
+class potato_msv_msaml: potato_msv_msamag {// SAM Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RADIO_LR};
+};
+class potato_msv_mtrl: Fic_Spotter { // Mortar Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER,RADIO_LR};
+};
+class potato_msv_mtrg: potato_msv_rifleman {// Mortar Gunner
+  MORTAR_GEAR("O_Mortar_01_weapon_F")
+};
+class potato_msv_mtrag: Fic_Spotter {// Assistant Mortar
+  MORTAR_GEAR("O_Mortar_01_support_F")
+};
+class spotter_F: Fic_Spotter {// Spotter
+  weapons[] = {SPOTTER};
+  magazines[] = {SPOTTER_MAG,BASE_GRENADES};
+  items[] += {RADIO_MR,"ACE_ATragMX","ACE_Kestrel4500","ACE_RangeCard"};
+  linkedItems[] += {LEADER_LINKED};
+  attachments[] = {"optic_KHS_blk","bipod_02_F_blk"};
+};
+class potato_msv_sniper: spotter_F {// Sniper
+  weapons[] = {SNIPER};
+  magazines[] = {SNIPER_MAG,BASE_GRENADES};
+  items[] = {BASE_TOOLS,"ACE_RangeCard"};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"optic_KHS_blk","bipod_02_F_blk"};
+  opticChoices[] = {};
+};
+class potato_msv_pilot {// Pilot
+  uniform[] = {"U_O_PilotCoveralls"};
+  vest[] = {};
```

Comment:
Wow, I just looked at it, that clipping is rough, but the pilots are going to want some personal protection when the rounds go right through the windshield take a look at the worker's coveralls and any vest (it should look roughly the part)
`U_C_WorkerCoveralls`
`V_TacVestIR_blk`